Adding week 6 notes

ecglazer opened the pull request.

ecglazer committed:
Adding week 6 notes

jkingslake commented on notes/week-6-notes.md:
```diff
@@ -0,0 +1,156 @@
+
+Driving stress drives ice flow.
+
+Imagine a column of ice. The pressure at each depth $z$ can be expressed as $P = \rho g z \partial y$, where:
```
```suggestion
Imagine a column of ice. The pressure at each depth $z$ can be expressed as $P = \rho g z$, where:
```
We can multiple by $\delta y$ later when we times by the area to get $F$

jkingslake commented on notes/week-6-notes.md:
```diff
@@ -0,0 +1,156 @@
+  - $z$ is the depth beneath the surface
+  - $\partial y$ is the thickness of the ice column
+
+Therefore, the pressure gradient with depth is $\frac{dP}{dz} = \rho g \partial y$.
```
```suggestion
Therefore, the pressure gradient with depth is $\frac{dP}{dz} = \rho g $.
```
sam reasoning as above

jkingslake commented on notes/week-6-notes.md:
```diff
@@ -0,0 +1,156 @@
+  - $\rho$ is the density of ice ( $~9.17 kg/m^3$)
+  - $g$ is gravitational acceleration ( $~9.8 m/s^2$)
+  - $z$ is the depth beneath the surface
+  - $\partial y$ is the thickness of the ice column
```
```suggestion
```

jkingslake commented on notes/week-6-notes.md:
```diff
@@ -0,0 +1,156 @@
+
+To calculate the force acting on one side of the ice column (blue arrow), with thickness H, we can consider that $F = P \times A = P(z) \times H \partial y$. Then, we can integrate over the height of the column to get the total force. We will call the force on the left-hand side of the column $F_1$:
+
+$F_1 = \int_0^H P(z) \partial y dz$
```
```suggestion
$F_1 = \int_0^H P(z) \delta y dz$
```
let's use $\delta$ for these small changes, not $\partial$

jkingslake commented on notes/week-6-notes.md:
```diff
@@ -0,0 +1,156 @@
+
+The more relevant quantity to consider here is not just the force acting on one side of the ice column, but rather the difference between the forces on either side. If we imagine that the ice column is part of an ice sheet with a sloping surface of grade $\frac{dH}{dx}$, and has thickness $\partial x$ in the x-direction, then the force acting on the opposite side of the ice column can be calculat ...
+
+$F_2 = \frac{\rho g \partial y H_2^2}{2} = \frac{\rho g \partial y H^2}{2} + \partial x \frac{d}{dx}(\frac{\rho g \partial y H^2}{2})$
```
```suggestion
$F_2  = \frac{\rho g \delta y H^2}{2} + \delta x \frac{d}{dx}(\frac{\rho g \delta y H^2}{2})$
```
i think there was an extra bit in this equation which wasnt needed

jkingslake commented on notes/week-6-notes.md:
```diff
@@ -0,0 +1,156 @@
+
+$F_2 = \frac{\rho g \partial y H_2^2}{2} = \frac{\rho g \partial y H^2}{2} + \partial x \frac{d}{dx}(\frac{\rho g \partial y H^2}{2})$
+
+$\Delta F = F_1 - F_2 = \frac{\rho g \partial y H^2}{2} - (\frac{\rho g \partial y}{2}H^2 + \partial \frac{d}{dx}(\frac{\rho g \partial y H^2}{2}))$
```
```suggestion
$\Delta F = F_1 - F_2 = \frac{\rho g \delta y H^2}{2} - (\frac{\rho g \delta y}{2}H^2 + \delta x \frac{d}{dx}(\frac{\rho g \partial y H^2}{2}))$
```

ecglazer committed:
Update notes/week-6-notes.md

makes sense, i noticed that the dimensional analysis didn't quite work out before

Co-authored-by: jkingslake <48723181+jkingslake@users.noreply.github.com>

ecglazer committed:
Update notes/week-6-notes.md

Co-authored-by: jkingslake <48723181+jkingslake@users.noreply.github.com>

ecglazer committed:
Update notes/week-6-notes.md

Co-authored-by: jkingslake <48723181+jkingslake@users.noreply.github.com>

ecglazer committed:
Update notes/week-6-notes.md

not the first (or last) time i've mixed up partial and delta symbols...

Co-authored-by: jkingslake <48723181+jkingslake@users.noreply.github.com>

ecglazer committed:
Update notes/week-6-notes.md

Co-authored-by: jkingslake <48723181+jkingslake@users.noreply.github.com>

ecglazer committed:
Update notes/week-6-notes.md

Co-authored-by: jkingslake <48723181+jkingslake@users.noreply.github.com>
